packages/wasm/wasm.2.0.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+maintainer: "Andreas Rossberg <rossberg@mpi-sws.org>"
+authors: "Andreas Rossberg <rossberg@mpi-sws.org>"
+homepage: "https://github.com/WebAssembly/spec"
+bug-reports: "https://github.com/WebAssembly/spec/issues"
+license: "Apache-2.0"
+dev-repo: "git+https://github.com/WebAssembly/spec.git"
+build: [
+  [make "-C" "interpreter" "opt" "unopt"]
+]
+install: [make "-C" "interpreter" "install"]
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "ocamlfind" {build}
+  "ocamlbuild" {build}
+  "dune" {>= "2.9"}
+]
+synopsis:
```

Comment:
```suggestion
available: arch != "arm32" & arch != "ppc64"
```

Due to:
```
#=== ERROR while compiling wasm.2.0.0 =========================================#
# context              2.1.2 | linux/ppc64 | ocaml-base-compiler.4.14.0 | pinned(https://github.com/WebAssembly/spec/archive/opam-2.0.0.zip)
# path                 ~/.opam/4.14/.opam-switch/build/wasm.2.0.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build make -C interpreter opt unopt
# exit-code            2
# env-file             ~/.opam/log/wasm-3246-4976a9.env
# output-file          ~/.opam/log/wasm-3246-4976a9.out
### output ###
# make: Entering directory '/home/opam/.opam/4.14/.opam-switch/build/wasm.2.0.0/interpreter'
# echo >_tags "true: bin_annot"
# echo >>_tags "true: debug"
# echo >>_tags "<{util,syntax,binary,text,valid,runtime,exec,script,host,main,tests}/*.cmx>: for-pack(Wasm)"
# ocamlbuild -lexflags -ml -cflags '-w +a-4-27-42-44-45-70 -warn-error +a-3' -I util -I syntax -I binary -I text -I valid -I runtime -I exec -I script -I host -I main -I tests -libs bigarray -quiet main.native
# + /home/opam/.opam/4.14/bin/ocamlopt.opt -c -w +a-4-27-42-44-45-70 -warn-error +a-3 -g -bin-annot -for-pack Wasm -I text -I valid -I util -I tests -I syntax -I script -I runtime -I main -I host -I exec -I binary -o text/lexer.cmx text/lexer.ml
# /opam-tmp/camlasm11a009.s: Assembler messages:
# /opam-tmp/camlasm11a009.s:97170: Error: operand out of range (0x00000000000080ec is not between 0xffffffffffff8000 and 0x0000000000007ffc)
# /opam-tmp/camlasm11a009.s:97172: Error: operand out of range (0x00000000000080d8 is not between 0xffffffffffff8000 and 0x0000000000007ffc)
# /opam-tmp/camlasm11a009.s:97231: Error: operand out of range (0x000000000000800c is not between 0xffffffffffff8000 and 0x0000000000007ffc)
# /opam-tmp/camlasm11a009.s:97235: Error: operand out of range (0x0000000000008000 is not between 0xffffffffffff8000 and 0x0000000000007ffc)
# File "text/lexer.ml", line 1:
# Error: Assembler error, input left in file /opam-tmp/camlasm11a009.s
# Command exited with code 2.
# make: *** [Makefile:70: main.native] Error 10
# rm _tags
# make: Leaving directory '/home/opam/.opam/4.14/.opam-switch/build/wasm.2.0.0/interpreter'
```
and
```
#=== ERROR while compiling wasm.2.0.0 =========================================#
# context              2.1.2 | linux/arm32 | ocaml-base-compiler.4.14.0 | pinned(https://github.com/WebAssembly/spec/archive/opam-2.0.0.zip)
# path                 ~/.opam/4.14/.opam-switch/build/wasm.2.0.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build make -C interpreter opt unopt
# exit-code            2
# env-file             ~/.opam/log/wasm-2692-cf35b8.env
# output-file          ~/.opam/log/wasm-2692-cf35b8.out
### output ###
# make: Entering directory '/home/opam/.opam/4.14/.opam-switch/build/wasm.2.0.0/interpreter'
# echo >_tags "true: bin_annot"
# echo >>_tags "true: debug"
# echo >>_tags "<{util,syntax,binary,text,valid,runtime,exec,script,host,main,tests}/*.cmx>: for-pack(Wasm)"
# ocamlbuild -lexflags -ml -cflags '-w +a-4-27-42-44-45-70 -warn-error +a-3' -I util -I syntax -I binary -I text -I valid -I runtime -I exec -I script -I host -I main -I tests -libs bigarray -quiet main.native
# + /home/opam/.opam/4.14/bin/ocamlopt.opt -c -w +a-4-27-42-44-45-70 -warn-error +a-3 -g -bin-annot -for-pack Wasm -I text -I valid -I util -I tests -I syntax -I script -I runtime -I main -I host -I exec -I binary -o text/lexer.cmx text/lexer.ml
# /opam-tmp/camlasmd158c0.s: Assembler messages:
# /opam-tmp/camlasmd158c0.s:47874: Error: value of 102622 too large for field of 2 bytes at 79974
# /opam-tmp/camlasmd158c0.s:47875: Error: value of 102622 too large for field of 2 bytes at 79976
# /opam-tmp/camlasmd158c0.s:47877: Error: value of 102491 too large for field of 2 bytes at 79980
[...]
# /opam-tmp/camlasmd158c0.s:48531: Error: value of 102217 too large for field of 2 bytes at 81174
# File "text/lexer.ml", line 1:
# Error: Assembler error, input left in file /opam-tmp/camlasmd158c0.s
# Command exited with code 2.
# make: *** [Makefile:70: main.native] Error 10
```

Reply:
Yes, this is rather annoying. Have a PR [in flight](https://github.com/WebAssembly/spec/pull/1482) to refactor the lexer to avoid the table overflow. Any idea why this only is an issue on these two architectures?

Reply:
No idea. But only the tests are failing, you can keep them on and failing if you prefer (to warn potential users) or disable the architectures altogether. Here I am just suggesting to disable the tests